components/channel/components/commands-button.tsx

```
@@ -0,0 +1,85 @@
+  const {
+    theme: {
+      colors: { accent_blue, grey },
+      messageInput: { commandsButton },
+    },
+  } = useTheme();
+  const appTheme = useAppTheme();
+
+  return (
+    <Pressable onPress={handleOnPress} style={[commandsButton]} testID='commands-button'>
+      <FontAwesomeIcon
+        color={suggestions && suggestions.data.some((suggestion) => isSuggestionCommand(suggestion))
```

Biome (1.9.4)

[error] Change to an optional chain.

Unsafe fix: Change to an optional chain.


(lint/complexity/useOptionalChain)